enginetest/queries/queries.go

```diff
@@ -2723,6 +2723,38 @@ var QueryTests = []QueryTest{
 		Query:    "SELECT i FROM mytable ORDER BY i LIMIT 1 OFFSET 1;",
 		Expected: []sql.Row{{int64(2)}},
 	},
+	{
```

Comment:
seems like good subquery and single relation coverage! might be worth adding join tests, root and subscopes, might behave differently